applications/fluid_structure_interaction/pressure_wave/application.h

```diff
@@ -622,9 +619,8 @@ class Application : public ApplicationBase<dim, Number>
     param.spectral_radius                      = 0.8;
     param.solver_info_data.interval_time_steps = OUTPUT_SOLVER_INFO_EVERY_TIME_STEPS;
 
-    param.triangulation_type = TriangulationType::Distributed;
-    param.mapping            = MAPPING_TYPE;
-    param.degree             = degree;
+    param.grid.triangulation_type = TriangulationType::Distributed;
+    param.grid.mapping_degree     = MAPPING_DEGREE;
```

Comment:
`this->structure_param.degree` would mean isoparametric mapping. In this example, a global variable `MAPPING_DEGREE` is used, which has the advantage that it can be used for all fields (this is an FSI problem and not a single field problem). Of course, one could also use something like `this->structure_param.degree` for all occurrences, but this would imply implicit knowledge about the order in which the parameters are initialized for the different fields (fluid, structure, ALE). This order can currently not be deduced from the `Application` class, but only in combination with the `Driver`, which is clearly a shortcoming of the current design and one of the aspects that I mean with "I encountered further ideas of refactoring during this PR, but these are best addressed in separate PRs.".

Reply:
I see, now with the parameter also for velocity it makes sense to me.